.github/checkgroup.yml

```diff
@@ -108,7 +108,7 @@ subprojects:
       - "!*.md"
       - "!**/*.md"
     checks:
-      - "make-doctest (fabric)"
+      - "docs-checks (fabric, doctest)"
```

Comment:
Where is the configuration that the check is not required?

Reply:
It is not added to the group check =)